lib/decorateWithMetrics.js

```diff
@@ -74,30 +74,42 @@ function getMetrics() {
             const avg = (total / calls.length) || 0;
             const max = _.max(calls, 'duration').duration || 0;
             const min = _.min(calls, 'duration').duration || 0;
+            const lastCall = _.max(calls, 'endTime');
 
             return [methodName, {
                 methodName,
                 total,
                 max,
                 min,
                 avg,
+                lastCall,
                 calls,
             }];
         })
         .object() // Create a map like methodName -> StatSummary
         .value();
 
     const totalTime = sum(_.values(summaries), 'total');
-    const averageTime = (totalTime / _.size(summaries)) || 0;
```

Comment:
Removed this as it doesn't provide anything useful 
it's just `total time spent / number of decorated methods` 

Added `lastCall` per each summary and `lastCompleteCall` that is the most
recent operation of all that ended 

I use this to track when we're done making operations with Onyx e.g. 
- when we're done reading from Onyx during init 
- when we're done reading during after switching to a new chat